[Regression] Fails to build on both ppc64le and s390x

I recently tried to update the ```osconfig``` package in SUSE Linux Enterprise Server 12 and 15 to their latest upstream versions.

Unfortunately, the package fails to build on both ppc64le and s390x. In both cases, the failure is the same:

```
[   64s] + go build '-ldflags=-s -w -X main.version=20201117.00-1.6.1' -mod=vendor -o google_osconfig_agent
[   74s] # cos.googlesource.com/cos/tools.git/src/pkg/cos
[   74s] vendor/cos.googlesource.com/cos/tools.git/src/pkg/cos/env_reader.go:65:82: cannot use c.uname.Release[:] (type []uint8) as type []int8 in argument to charsToString
[   74s] error: Bad exit status from /var/tmp/rpm-tmp.SZQQ4F (%build)
```

I have not bisected the issue yet but earlier versions were not affected, so it's a regression.

--- a/packages/cos.go
+++ b/packages/cos.go
@@ -12,6 +12,10 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
+// Only build for linux but not s390x ppc64le ppc64.
+
+// +build linux,!s390x,!ppc64le,!ppc64
+
 package packages
 
 import (

--- a/packages/cos_stub.go
+++ b/packages/cos_stub.go
@@ -1,0 +1,25 @@
+//  Copyright 2021 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// Stub for s390x ppc64le ppc64 linux builds.
+
+// +build linux
+// +build s390x ppc64le ppc64
+
+package packages
+
+// InstalledCOSPackages is a stub for unsupported architectures.
+func InstalledCOSPackages() ([]PkgInfo, error) {
+	return nil, nil
+}

--- a/packages/cos_test.go
+++ b/packages/cos_test.go
@@ -12,6 +12,8 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
+// +build linux,!s390x,!ppc64le,!ppc64
+
 package packages
 
 import (

